lta/picker.py

```diff
@@ -16,7 +16,10 @@
 
 EXPECTED_CONFIG = [
     "FILE_CATALOG_REST_URL",
+    "HEARTBEAT_PATCH_RETRIES",
+    "HEARTBEAT_PATCH_TIMEOUT_SECONDS",
```

Comment:
The only comment I have on these two variables is that they might not need to be configurable.  A sensible default is probably good enough.